### PR TITLE
Removal of extra dimensions on the cloud watch alarm.

### PR DESCRIPTION
**Pull request by lenuhc**
* Removed InstanceID and Endpoint since they are interfering with the Orchestrator when it creates the alarm

### CODE REVIEW

File: templates/aem-tools/content-healthcheck.py.epp [hunk 1]
```diff
@@ -160,15 +160,15 @@ def check_random_resource(resource_list, resource_type):
     return process_response(submit_request((resource_list[0])))
 
 # Sends cutom metric to cloudwatch with the following dimensions:
-# instance_id which is the distpacher Id, PairInstanceId which is the publisher Id, and the Endpoint which is the actual content path
+# PairInstanceId which is the publisher Id
 # Prints the status of the the current content path
 def send_to_cloudwatch(status):
     dispatcher_id = get_instance_id()
```
Comment (michaeldiender-shinesolutions) [inline templates/aem-tools/content-healthcheck.py.epp]: does this get used?

File: templates/aem-tools/content-healthcheck.py.epp [hunk 2]
```diff
@@ -160,15 +160,15 @@ def check_random_resource(resource_list, resource_type):
...
     publisher_id = get_instance_tag('PairInstanceId')
     cw = boto3.client('cloudwatch', region_name=region)
     namespace = get_instance_tag('aws:cloudformation:stack-name')
     name = 'contentHealthCheck'
-    dimensions = [{'Name': 'InstanceId','Value': dispatcher_id},{'Name': 'PairInstanceId','Value': publisher_id},{'Name': 'Endpoint','Value': endpoint}]
+    dimensions = [{'Name': 'PairInstanceId','Value': publisher_id}]
     metric_data = { 'MetricName': name, 'Dimensions': dimensions, 'Value': status, 'Unit': 'Count' }
     try:
         cw_response = cw.put_metric_data(Namespace=namespace, MetricData=[ metric_data ])
```
Comment (michaeldiender-shinesolutions) [inline templates/aem-tools/content-healthcheck.py.epp]: does the message make sense? 'message = 'The content path ' + endpoint + ' in the publisher instance ' + publisher_id + ' is '

File: templates/aem-tools/content-healthcheck.py.epp (hunk 1 above)
Comment (lenuhc) [inline templates/aem-tools/content-healthcheck.py.epp]: No. Let me remove it.

File: templates/aem-tools/content-healthcheck.py.epp (hunk 2 above)
Comment (lenuhc) [inline templates/aem-tools/content-healthcheck.py.epp]: Not by itself. But combined with this: log.info('%s healthy.', message) I believe it does.

**Pull request merged**